src/scalecube-transport/api/TransportInterface.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import { TransportRequest, ProviderConfig } from './types';
+import { ProviderInterface } from './ProviderInterface';
+import { Observable } from 'rxjs';
+
+export interface TransportInterface {
```

Comment:
I think it will look better just Transport

Reply:
Why? We have a class Transport and an interface TransportInterface. I thought it was our common approach.